blackbox/docs/release_notes/1.1.3.txt

```diff
@@ -0,0 +1,73 @@
+.. _version_1.1.3:
+
+=============
+Version 1.1.3
+=============
+
+Released on 2017/05/09.
+
+.. NOTE::
+
+   If you are upgrading a cluster, you must be running CrateDB
+   :ref:`version_1.0.2` or higher before you upgrade to 1.1.3.
+
+   If you want to perform a :ref:`cluster_upgrade`, your current CrateDB
+   version number must be :ref:`version_1.1.2`. Any upgrade from a version
+   prior to this will require a full cluster restart.
+
+.. WARNING::
+
+   Before upgrading, you should `back up your data`_.
+
+.. _back up your data: https://crate.io/a/backing-up-and-restoring-crate/
+
+Fixes
+=====
+
+ - Admin UI improvements.
+
+ - Improved the accuracy of results if arithmetic operators +,-,*,/ are used
+   in expressions that contain only float-type values.
+
+ - Fixed ``COPY FROM`` to be able to copy data into a partitioned table with a
+   generated column as both the primary key and the partitioned column.
+
+ - ``NullPointerException`` thrown when selecting ``routing_hash_function``
```

Comment:
I think this needs to be reworded as this makes it sound as if we *now* throw a NPE when selecting from sys.shards - this fix was to fix this from happening.